zubhub_frontend/zubhub/src/assets/js/styles/components/notification/NotificationStyles.js

```diff
@@ -0,0 +1,91 @@
+const styles = theme => ({
+    notificationStyle: {
+        display: 'flex',
+        borderRadius: '5px',
+        width: '340px',
```

Comment:
On the mobile view, the notifications don't take up the entire width of the screen because the width is pre-defined here. You can use [breakpoints](https://mui.com/customization/breakpoints/) to determine when you should set the width.